metasyncontrib/disclosure/base.py

```diff
@@ -30,6 +30,13 @@ def fit(cls, series, *args, n_avg: int = 11, **kwargs) -> BaseDistribution:
         """Fit constant distributions with disclosure control rules in place."""
         pl_series: pl.Series = cls._to_series(series)
 
+        # NB: dominance rule ensures that constant distribution is essentially never
+        # allowed under formal disclosure control. Always return default distribution.
+        apply_dominance_rule: bool = True
```

Comment:
I think we can remove the old code that won't be reached now. We have git in case we would like to get back to the old way.

Reply:
yeah I was preparing for a potential "apply dominance" parameter in `DisclosurePrivacy()`